objects/src/accounts/mod.rs

```diff
@@ -114,6 +111,11 @@ impl Account {
         &self.vault
     }
 
+    /// Returns a mutable reference to the vault of this account.
+    pub fn vault_mut(&mut self) -> &mut AccountVault {
+        &mut self.vault
+    }
```

Comment:
Seems like this is used only for testing/mock purposes - should we put it behind `testing` feature flag?